libs/commands/version/README.md

```diff
@@ -293,9 +293,10 @@ lerna version --force-publish=package-2,package-4
 lerna version --force-publish
 ```
 
-When run with this flag, `lerna version` will force publish the specified packages (comma-separated) or all packages using `*`.
+When run with this flag, `lerna version` will force publish the specified packages (comma-separated) or all packages using `*`. This option is ignored when
```

Comment:
>This option is ignored when

Is this meant to be here? It looks like this is stated below in the NOTE

Reply:
Yes that was a mistake. Should be ok now.